src/Entity/InformationProductType.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\InformationProductTypeRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=InformationProductTypeRepository::class)
+ */
+class InformationProductType extends Entity
```

Comment:
Just tested, and this can not be call `type` 

"Could not load type \"App\\Entity\\InformationProductType\": class does not implement \"Symfony\\Component\\Form\\FormTypeInterface\"."